Naisten-Linja/naisten-linja-public-website
# Fix/reference to internal page

A fix to a problem where internal pages referenced in content/service boxes weren't referenced correctly. The most likely cause of this is that I updated the contentful source plugin, and didn't catch those references when doing the migration. 

## How to test?

1. Open the page
2. Find some content boxes/service boxes 
3. See that boxes with internal links have the links too

## Screenshots
<img width="1200" alt="Screenshot 2021-08-04 at 7 16 56" src="https://user-images.githubusercontent.com/28345294/128121410-2f91a0bd-1b28-4d39-9310-28270b93a405.png">
